packages/wonder-blocks-grid/components/grid.js

```diff
@@ -161,8 +162,10 @@ export default class Grid extends React.Component<
     }
 
     render() {
+        // eslint-disable-next-line no-unused-vars
```

Comment:
All of these changes are super-annoying. I had to apply styling to the Grid component as having an un-styled flexbox element was messing up the layout. This will all get removed when we can use React fragments.

Reply:
I don't follow.  Is there supposed to be a `style` prop within `otherProps`?

Reply:
I see... `GridContext` is defined below.  Why isn't it in its own file?

Reply:
We really don't want anyone using it - it's only purpose is to create the React Context (which we will need to rewrite when we upgrade to React 16+)